components/Card.js

```diff
@@ -5,13 +5,15 @@ import clsx from 'clsx';
 
 function Card({ backgroundImageSource, backgroundImageAltText, children }) {
   return (
-    <div className="card filter brightness-105 shadow-xl image-full">
+    <div className="card filter brightness-105 shadow-xl">
       {backgroundImageSource && (
         <figure>
           <Image
             alt={backgroundImageAltText}
-            layout="fill"
+            height="250"
+            layout="fixed"
```

Comment:
Hey @ozer619. So I played around more after we talked and `layout="responsive"` seems to do the trick of keeping the aspect ratio here. Seems like an appropriate term! 😁